src/dapla_metadata/variable_definitions/generated/vardef_client/docs/Person.md

```diff
@@ -0,0 +1,30 @@
+# Person
+
+
+## Properties
+
+Name | Type | Description | Notes
+------------ | ------------- | ------------- | -------------
+**code** | **str** |  | 
+**name** | **str** |  | 
+
+## Example
+
+```python
+from vardef_client.models.person import Person
+
+# TODO update the JSON string below
```

Comment:
Where does all the TODO's come from? Or why 😀

Reply:
They're generated along with everything else. I believe it's a note to the user of the code example to input their own data.